app/assets/javascripts/client/inventories/AddInventoryUsersCtrl.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient').controller('AddInventoryUsersCtrl', AddInventoryUsersCtrl);
+
+  AddInventoryUsersCtrl.$inject = ['$scope', 'InventoryInvitable', 'InventoryParticipant'];
+  function AddInventoryUsersCtrl($scope, InventoryInvitable, InventoryParticipant) {
+    var vm = this;
+    vm.reloadInvitables = function() {
```

Comment:
`loadInvitables` may be a better name for this since we're still loading the invitables from the system, even after we hit the page for the first time.